spring-cloud-stream-binder-kinesis/src/main/java/org/springframework/cloud/stream/binder/kinesis/properties/KinesisBinderConfigurationProperties.java

```diff
@@ -44,6 +44,9 @@
 
 	private int minShardCount = 1;
 
+	/** Enables the usage of Amazon KCL/KPL libraries for all message consumption and production. */
```

Comment:
This is not a JavaDoc.
Please, consider to have one more line in this description:
```suggestion
	/** 
	* Enables the usage of Amazon KCL/KPL libraries for all message consumption and production. 
	*/
```